src/index.ts

```diff
@@ -4,4 +4,7 @@ export * from './platform/stub-setup';
 export * from './platform/smart-home-app';
 export * from './platform/device-manager';
 export * from './platform/execute';
+/**
```

Comment:
It seems like a number of these changes are also in #39. Once that's added, it'll be good to rebase this, which may shrink the diff.

Reply:
Noted.  If you want to isolate just the CLI changes for now, I squashed all these changes so viewing 383d230 in GitHub will ignore the overlapping changes from #39 . (though this will change as I update this PR)

Reply:
Ack